python/pyarrow/_dataset_parquet.pyx

```diff
@@ -236,9 +236,9 @@ class RowGroupInfo:
 
     Parameters
     ----------
-    id : the group id.
-    metadata : the rowgroup metadata.
-    schema : schema of the rows.
+    id : the group id
```

Comment:
Similar to before, the place after the colon is for the type, so the description should be moved to the next line I think.

Reply:
Same response as above too, though I think I can fix this in this PR.